subprojects/gradle-plugin/src/main/java/org/spongepowered/gradle/vanilla/internal/repository/IvyModuleWriter.java

```diff
@@ -155,11 +155,26 @@ private void writeDependencies(final Set<GroupArtifactVersion> dependencies) thr
     }
 
     private void writeDependency(final GroupArtifactVersion dep) throws XMLStreamException {
-        this.writer.writeEmptyElement("dependency");
+        boolean hasClassifier = dep.classifier() != null;
```

Comment:
If you change the ivy file generation, you need to change the meta version in `MinecraftResolver.METADATA_VERSION`